xapian-applications/omega/omegatest

```diff
@@ -402,6 +402,18 @@ testcase "A${tab}AZ${tab}Z" A=1 A=2 A=3 AZ=1 AZ=2 Z=xxx
 testcase "${tab}abc" =1 abc=1
 testcase "${tab}abc${tab}def" =1 abc=1 def=7
 
+# Feature tests for $highlight{}.
+printf '$highlight{$cgi{text},$cgi{list}}' > "$TEST_TEMPLATE"
+testcase 'A <b style="color:black;background-color:#ffff66">list</b> of <b style="color:black;background-color:#99ff99">words</b>' P=text list="list${tab}words" text="A list of words" open="<b>" close="</b>"
```

Comment:
`P` also seems to be unused (or is it implicitly required for the template to work?)